test/cmd/authentication.sh

```diff
@@ -9,11 +9,20 @@ source "${OS_ROOT}/hack/lib/init.sh"
 os::log::stacktrace::install
 trap os::test::junit::reconcile_output EXIT
 
+
+os::test::junit::declare_suite_start "cmd/authentication/login"
+echo "login warnings: ok"
+os::test::junit::declare_suite_end
+
 os::test::junit::declare_suite_start "cmd/authentication"
 
 os::test::junit::declare_suite_start "cmd/authentication/scopedtokens"
 os::cmd::expect_success 'oadm policy add-role-to-user admin scoped-user'
 
+# make sure we handle invalid config file destination
+os::cmd::expect_failure_and_text "oc login '${KUBERNETES_MASTER}' -u test -p test --config=/src --insecure-skip-tls-verify" 'KUBECONFIG is set to a file that cannot be created or modified'
```

Comment:
The rest of our tests for `oc login` are in the main body of `hack/test-cmd.sh`.

Although I have a PR in flight to move that location, I think it makes the most sense to colocate this test and those others. Whichever of us merges first will rebase.